### PR TITLE
[20.01] Fixes for get_estimated_display_viewport

### PR DESCRIPTION
**Pull request by mvdbeek**
Map objects cannot be unpacked, decrease log level, use context managers
for opening files.

Silences https://sentry.galaxyproject.org/sentry/main/issues/517474/:
```
line (##sequence-regionSL4.0ch07	1	67883646
) uses an unsupported ##sequence-region definition.
```
and similar.

### CODE REVIEW

File: lib/galaxy/datatypes/interval.py [hunk 1]
```diff
@@ -167,40 +167,39 @@ def get_estimated_display_viewport(self, dataset, chrom_col=None, start_col=None
...
-                        pass
-                # Make sure we are at the next new line
-                readline_count = VIEWPORT_MAX_READS_PER_LINE
-                while line.rstrip('\n\r') == line:
-                    assert readline_count > 0, Exception('Viewport readline count exceeded for dataset %s.' % dataset.id)
-                    line = fh.readline(VIEWPORT_READLINE_BUFFER_SIZE)
+            with open(dataset.file_name) as fh:
+                for line in util.iter_start_of_line(fh, VIEWPORT_READLINE_BUFFER_SIZE):
+                    # Stop if at end of file
                     if not line:
```
Comment (nsoranzo) [inline lib/galaxy/datatypes/interval.py]: Is this check still needed, given you are using a for loop?

Comment (mvdbeek) [reply lib/galaxy/datatypes/interval.py]: it's not, I dropped it in the other function~s~ and forgot this one, thanks!

**Comment by mvdbeek** [comment]
@nsoranzo I restored the map unpacking in the last commit here.

**Comment by nsoranzo** [comment]
> @nsoranzo I restored the map unpacking in the last commit here.

Thanks, I've pushed 2 other small follow-ups to #9416.

**Review by nsoranzo — APPROVED** [review]

**Pull request merged**